### PR TITLE
Android logging module

### PR DESCRIPTION
**Pull request by vico1103**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -163,16 +169,17 @@ android {
...
-    compile (project(':react-native-maps')){
-        exclude group: 'com.google.android.gms', module: 'play-services-base'
-        exclude group: 'com.google.android.gms', module: 'play-services-maps'
-    }
-    compile 'com.google.android.gms:play-services-base:11.8.0'
-    compile 'com.google.android.gms:play-services-maps:11.8.0'
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    implementation project(':react-native-svg')
```
Comment (mrtnzlml) [inline android/app/build.gradle]: _Note to me: this must be removed (rebase it again 🙄). I will do it._

File: android/build.gradle [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 
 buildscript {
+    apply from: "dependencies.gradle"
+
     repositories {
         // this line will fix Could not find com.android.tools.build:gradle:3.0.1
         // https://stackoverflow.com/a/47456492/3135248
         google()
         jcenter()
+        google()
```
Comment (mrtnzlml) [inline android/build.gradle]: Brilliant. This is not rebased at all... 😄

Comment (vico1103) [reply android/build.gradle]: I rebased it, but i didn't see this

Comment (mrtnzlml) [reply android/build.gradle]: @vico1103 You didn't use the latest master so I have so many conflicts to resolve now. Basically, nothing changed from the last PR if I am being honest...

Comment (mrtnzlml) [reply android/build.gradle]: @vico1103 Leave it, I'll try to merge it. I will let you know in case of any complications.

Comment (vico1103) [reply android/build.gradle]: I use master from 1 pm

**Comment by mrtnzlml** [comment]
Ok, rebased again, squashed, integrated.

**Pull request closed without merging**